### ISSUE TITLE
start_frame in Method I

### ISSUE THREAD

**Issue by mirakuepper** [JuPedSim/jpscore#776]
Current date   : Fri May 15 11:33:50 2020
Version        : 0.8.4
Compiler       : g++ (8.4.0)
Commit hash    : v0.8.4-968-gaa0b4ed8-dirty
Commit date    : Mon May 11 14:51:30 2020

JPSreport creates a segmentation fault (core dumped) when setting 
`start_frame` in Method I to any value. Only if it is set to "None", it works. 

`stop_frame` can be set to different values and it still works (as long as start_frame = "None").

I found this problem when I tried to use Method I on the demo-files for the corridor. (jpsreport/demos/corridor). 
[issue.zip](https://github.com/JuPedSim/jpscore/files/4633673/issue.zip)

**Comment by mirakuepper** [comment]
Maybe I should add, that `start_frame` in Method D works (at least for the values I tested).

For me this is not a critical issue. I just found it by chance.

**Comment by JetteSchumann** [comment]
This issue was solved by PR #808. Method I is no longer available. Instead, you can configure Method D to use the whole geometry as a measurement area by:
```xml
<method_D enabled="true">
      <measurement_area id="2" start_frame="10" stop_frame="12"/>
      <cut_by_circle enabled="false" radius="1.0" edges="10"/>
      <global_IFD enabled="true"/>
  </method_D>
```
